tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -154,16 +161,41 @@ protected void performPostFinish()
     {
         super.performPostFinish();
 
+        List<IProject> finalProjects = new ArrayList<IProject>();
+
         final NewLiferayPluginProjectOp op = element().nearest( NewLiferayPluginProjectOp.class );
+
         final IProject project = CoreUtil.getProject( op.getFinalProjectName().content() );
 
-        try
+        ElementList<ProjectName> projectNames = op.getProjectNames();
```

Comment:
Lets just use the new projectNames property only, so there is no need for the if/else... lets have both maven and sdk projects use the new projectNames property.